.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     name: Create cache for all steps
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repository
+      - name: Checkout code
```

Comment:
Why? 😄

Reply:
Because this is how we write title in 99%, this was exception

Reply:
I'm wondering if we even need these titles :D But yup, let is stay like everywhere. good one!